version.json

```diff
@@ -1,4 +1,4 @@
 {
-    "version": "4.5.0",
+    "version": "4.6.0",
```

Comment:
Please update the `4.6.0` references to `4.5.0`. As this will be merge into `4.5`